`SpanGroup(s)`-related optimizations (#11380)
explosion/spaCy: website/docs/api/spangroup.md

> new_group = doc.spans["errors"].copy()
> ```

| Name        | Description                                                                                                     |
| ----------- | --------------------------------------------------------------------------------------------------------------- |

> | `doc`       | New reference document to which the copy is bound. Defaults to `None`. ~~Optional[Doc]~~                        |
> | **RETURNS** | A copy of the `SpanGroup` object, bound to either its original reference document or the new one. ~~SpanGroup~~ |

Reviewer: Better to avoid "new" because it's not necessarily new and we're not creating any docs here. I think it's also better to avoid "reference doc" because that may be confusing vs. `Example`.

```suggestion
| `doc`       | The document to which the copy is bound. Defaults to `None` for the current doc. ~~Optional[Doc]~~                        |
| **RETURNS** | A copy of the `SpanGroup` object. ~~SpanGroup~~ |
```

(and then reformat)
